### PR TITLE
more fixes

### PR DESCRIPTION
**Pull request by odinuv**
fix: generate destination in manifest when unnamed default bucket is used (filename is still not valid table name)

fix: handle parentKey which is not named but present
fix: handle parentKey disable

### CODE REVIEW

File: src/MissingTableHelper.php [hunk 1]
```diff
@@ -59,13 +59,21 @@ private static function fillMissingTableMapping(
                     $primaryKey[] = $item['mapping']['destination'];
                 }
             } elseif ($item['type'] === 'table') {
+                if (empty($item['parentKey'])) {
+                    $parentKeyParam = ['destination' => $name . '_pk'];
```
Comment (MiroCillik) [inline src/MissingTableHelper.php:63]: Takze ked v tej child tabulke `parentKey` nie je nastaveny, tak sa pouzije "default" nazov parent sloupce + '_pk'. To je asi v pohode teda. Je tam na to nejaky test?

Comment (odinuv) [reply src/MissingTableHelper.php:63]: jop, tutok https://github.com/keboola/generic-extractor/pull/117/files#diff-6b3b53e020ad3e45990775eb05699f72R544

Comment (odinuv) [reply src/MissingTableHelper.php:63]: jo v ty dokumentaci je to nejakej renonc

Comment (odinuv) [reply src/MissingTableHelper.php:63]: issue jsem zalozil

**Comment by odinuv** [comment]
no nic, jdu na to, doufam, ze uz je to fakt vsechno, uz mi to leze krkem

**Pull request merged**